src/actions/console-message-actions.js

```diff
@@ -0,0 +1,45 @@
+import createHistoryItem from "../tools/create-history-item";
+
+export function evalTypeConsoleError(evalType) {
```

Comment:
this file is just wrappers to make adding things to the console easier. this could use a refactor, but out of scope for this PR